operations/configuration_management/Enable_Ansible_Profiling.md

```diff
@@ -1,26 +1,67 @@
 # Enable Ansible Profiling
 
-Ansible tasks and playbooks can be profiled in order to determine execution times and single out poor performance in runtime. The default Configuration Framework Service \(CFS\) `ansible.cfg` in the `cfs-default-ansible-cfg` ConfigMap does not enable these profiling tools. If profiling tools are desired, modify the default Ansible configuration file to enable them.
+Ansible tasks and playbooks can be profiled in order to single out poor performance in runtime.
+Profiling execution times is the most common use-case as this can directly affect boot times.
```

Comment:
The HPE Style guide dictates using "because" instead of "as" or "since" or "due to" (except for the construction "X is due to Y")
```suggestion
Profiling execution times is the most common use-case because this can directly affect boot times.
```